src/menu_display.c

```diff
@@ -61,7 +61,7 @@ void display_rules(void)
      printf(" Inputs to play the game\n");
     printf("  To request a card from the Draw pile, Press N or n. \n");
     printf("  To end turn, Press e or E. \n");
-    printf("  To discard a card - Say you want discard(RED,3), Press r3 or R3.\n");
+    printf("  To discard a card - Say you want discard(RED,3), Press r3.\n");
```

Comment:
We need to remove r3 instead of R3. 
The valid values are defined as:
static const char* CARD_VALUES[] = {   "R0", "R1", "R2","R3", "R4", "R5","R6", "R7", "R8","R9",
                                        "Y0", "Y1", "Y2","Y3", "Y4", "Y5","Y6", "Y7", "Y8","Y9",
                                        "B0", "B1", "B2","B3", "B4", "B5","B6", "B7", "B8","B9",
                                        "G0", "G1", "G2","G3", "G4", "G5","G6", "G7", "G8","G9" };

Reply:
Changed now, removed it by mistake.